qutebrowser/utils/version.py

```diff
@@ -347,7 +347,7 @@ def _chromium_version():
     if QWebEngineProfile is None:
         # This should never happen
         return 'unavailable'
-    profile = QWebEngineProfile()
+    profile = QWebEngineProfile.defaultProfile()
     ua = profile.httpUserAgent()
```

Comment:
Oh bother, we support changing the user agent by changing this don't we. I'll look at it again in a bit.